extensions/amp-font/amp-font.md

```diff
@@ -26,9 +26,16 @@ limitations under the License.
 
 # amp-font
 
-## Behavior
+## Do not use
+
+Use the [`font-display`](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display) descriptor to specify how to display a font face dependent on whether and when it is ready. 
```

Comment:
Can you provide a code snippet explaining how to cover the amp-font use cases with font-display?

Reply:
I'll look into it. @nainar @caroqliu do either of you have one/know someone who might have one ready to go?

Reply:
I don't unfortunately :(